airbyte-integrations/connectors/source-linkedin-pages/integration_tests/sample_config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "org_id": 12345678,
+  "credentials": {
+    "auth_method": "access_token",
+    "access_token": "drAL4JXBJ6v1qaU9iWargosra6ibiw0ZWEdMnC0ZizeD1gLRQP6u1pkQ"
```

Comment:
This is not a valid token right?

Reply:
no it is not, but i'll make it more obvious =)